cernopendata/config.py

```diff
@@ -390,6 +396,8 @@
     "CERNOPENDATA_MAX_DOWNLOAD_SIZE",
     200000000
 )
+#: Make download availalbe for Root files
+CERNOPENDATA_DISABLE_DOWNLOADS = False
```

Comment:
Please wrap in `os.environ.get()` so that one can influence it rapidly in a deployed containerised environment.

Example:

```
CERNOPENDATA_DISABLE_DOWNLOAD = os.environ.get(
     "CERNOPENDATA_DISABLE_DOWNLOADS",
     0
 )
```